Files/UserControls/StatusBarControl.xaml

```diff
@@ -151,14 +152,23 @@
                                 x:Uid="StatusBarControlLayoutMode"
                                 FontWeight="Medium"
                                 Text="Layout Mode" />
+
+                            <Button
+                                Content="Re-enable Adaptive Layout"
+                                x:Uid="StatusBarControlReEnableAdaptiveLayout"
+                                x:Name="ReEnableAdaptiveLayout"
+                                x:Load="{x:Bind FolderSettings.AdaptiveLayoutSuggestionOverriden, Mode=OneWay}"
+                                Command="{x:Bind FolderSettings.ReEnableAdaptiveLayout, Mode=OneWay}"
+                                HorizontalAlignment="Stretch"/>
```

Comment:
Do you think this button is redundant most of the time? Do you have any other ideas what we can do here?